doc/ref/debug.xml

```diff
@@ -457,15 +456,13 @@ in nanoseconds.
     in bytes allocated by the &GAP; memory manager since &GAP; started.
     </Description>
   </ManSection>
+  
   <ManSection>
-  <Var Name="memory_allocated"/>
-
-<Description>
-In the read-eval-print loop,
-<Ref Var="memory_allocated"/> stores the number of bytes of
-memoryallocated by the last completed statement.
-</Description>
-</ManSection>
+    <Var Name="memory_allocated"/>
+    <Description> In the read-eval-print loop, <Ref Var="memory_allocated"/>
+    stores the number of bytes of memory allocated by the last completed statement.
```

Comment:
This goes beyond a fix, so it's of course OK if you don't want to do this, but: in the `time` section you kept a ref to `memory_allocated`. Perhaps also add such a ref into the `memory_allocated`, pointing to `time`?

Reply:
@fingolfin why not, for the sake of reciprocity. Done.